### ISSUE TITLE
Такой as det to PP

### ISSUE THREAD

**Issue by ftyers** [UniversalDependencies/UD_Russian-SynTagRus#4]
Is this the analysis we want ? Should the `det` be `nsubj` here ? (and `DET` → `PRON`) ?

```
1       Такой   такой   DET     _       Case=Nom|Gender=Masc|Number=Sing        4       det     _       _
2       есть    быть    AUX     _       Aspect=Imp|Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin|Voice=Act      4       cop     _       _
3       в       в       ADP     _       _       4       case    _       _
4       Америке америка PROPN   _       Animacy=Inan|Case=Loc|Gender=Fem|Number=Sing    0       root    _       _
5       .       .       PUNCT   .       _       4       punct   _       _
```

**Comment by dan-zeman** [comment]
I think we want to change `det` to `nsubj` but keep the `DET` POS tag because this is an adjective-like demonstrative. It would be `PRON` under the v1 guidelines but based on the new approach, where form is more important, I would just say that this is a determiner promoted to the subject-head position because of ellipsis.

**Comment by ftyers** [comment]
Ok, that works. :)

**Comment by Kira-D** [comment]
@ftyers Thanks! This bug can produce many missed nsubj and nsubjpass. I fixed it.

Here is a list of DET:
det_set = {'такой', 'какой', 'всякий', 'некоторый', 'никакой', 'некий', 'сей', 'чей', 
           'какой-либо', 'какой-нибудь','кое-какой', 'весь', 'этот', 'тот',
           'мой', 'твой', 'ваш', 'наш',  'его', 'ее', 'их', 'свой'}

**Comment by olesar** [comment]
I agree with Dan. We apply this rule in many other cases as well.

**Comment by Kira-D** [comment]
Forgot to apply the fix to 2.1, but it's in dev brunch now
